recipes/splunk-opentelemetry-cpp/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+
+
+class SplunkOpentelemetryConan(ConanFile):
+    name = "splunk-opentelemetry-cpp"
+    license = "Apache 2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/signalfx/splunk-otel-cpp"
+    description = "Splunk's distribution of OpenTelemetry C++"
+    topics = ("opentelemetry", "observability", "tracing")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+    }
+    generators = "cmake", "cmake_find_package_multi"
+    requires = "opentelemetry-cpp/1.0.1"
+    exports_sources = "CMakeLists.txt"
+    short_paths = True
+    _cmake = None
+
+    def validate(self):
+        if self.settings.arch != "x86_64":
+            raise ConanInvalidConfiguration("Architecture not supported")
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def _remove_unnecessary_package_files(self):
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
```

Comment:
There's no `pkg_config` target declared in `package_info`

Reply:
Ah thanks, the `pkgconfig` deletion was a remnant from an earlier recipe version. Removed it now